.circleci/config.yml

```diff
@@ -50,6 +50,13 @@ jobs:
                 sleep 5
             done
 
+      - run:
+          name: run-audit-ci
+          command: |
+            nvm install-latest-npm
+            npm install --no-audit
```

Comment:
Does npm install usually do an audit? 
And if so what's the difference between that and audit-ci ?

Reply:
`npm install` does audit but doesn't return non-zero status if there is any audit errors.  `audit-ci` allows you specify a vulnerability level and returns non-zero status if there are vulnerabilities  found on that level or high.